testfm/content.py

```diff
@@ -20,7 +20,7 @@
 
 
 class Content(Base):
-    """Manipulates Foreman-maintain's content command"""
+    """Manipulates Satellite-maintain's content command"""
```

Comment:
can you check and replace f-m to s-m in class methods as well?

Reply:
still required: 
`testfm` ->  `testsm` (all imports in all files, repo name, ...)
`fm` -> `sm`

Reply:
I agree with @Gauravtalreja1 here, changing all imports in all files, repo name will result in numerous changes and would suggest proceeding with changes made for now.

Reply:
@Gauravtalreja1 so it's upon you whether you want to merge this.